README.md

```diff
@@ -23,27 +22,35 @@ Once you have the prerequisite installed, you can follow these steps:
     cd docs
     ```
 
-3. Preview the website using Quarto Preview:
+3. Instantiate the project environment:
+
+    ```bash
+    julia --project=. -e 'using Pkg; Pkg.instantiate()'
+    ```
+
+4. Preview the website using Quarto Preview:
 
     ```bash
     quarto preview
     ```
-This will launch a local server at http://localhost:4200/, which you can view in your web browser by navigating to the link shown in your terminal.
-Note: Avoid clicking links in the navbar while previewing locally because they will eventually lead to https links online!
 
-4. Render the website locally:
+    This will launch a local server at http://localhost:4200/, which you can view in your web browser by navigating to the link shown in your terminal.
+    Note: Avoid clicking links in the navbar while previewing locally because they will eventually lead to https links online!
+
+5. Render the website locally:
 
     ```bash
     quarto render
     ```
-This will render the full website in `_site` folder.
 
-It is also possible to render a single tutorial or `qmd` file without compiling the entire site. This is often helpful to speed up compilation when editing a single docs page. To do this, first `cd` to the file's folder, and run `quarto preview` or `quarto render`: 
+    This will render the full website in `_site` folder.
+
+    It is also possible to render a single tutorial or `qmd` file without compiling the entire site. This is often helpful to speed up compilation when editing a single docs page. To do this, first `cd` to the file's folder, and run `quarto preview` or `quarto render`: 
 
-```
-cd somedocs # This is the folder containing a docs `qmd` file. 
-quarto render index.qmd
-```
+    ```
+    cd somedocs # This is the folder containing a docs `qmd` file. 
```

Comment:
Yes, it still works but only as long as the env is instantiated. Quarto's Julia engine calls `julia --project=@.` so it will resolve to the top level env.

In the process of verifying this, though, I found another thing to add :)

Reply:
Done!